prismic-model/package.json

```diff
@@ -6,6 +6,7 @@
     "babel-cli": "^6.26.0",
     "babel-plugin-react": "^1.0.0",
     "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-env": "^1.6.1"
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-react": "^6.24.1"
```

Comment:
We're not using react here?

Reply:
Without it I get :

Error: Couldn't find preset "react" relative to directory "/blah/blah/blah/wellcomecollection.org/prismic-model"

when running yarn convert